automated/linux/cyclicdeadline/cyclicdeadline.sh

```diff
@@ -48,11 +50,24 @@ fi
 
 background_process_start bgcmd --cmd "${BACKGROUND_CMD}"
 
-"${binary}" -q -i "${INTERVAL}" -s "${STEP}" -t "${THREADS}" \
-	-D "${DURATION}" --json="${LOGFILE}"
+counter=0
+while [ ${counter} -lt ${ITERATIONS} ]; do
```

Comment:
Maybe ,try `for i in $(seq ${ITERATION})` so you don't have to maintain the counter.

Reply:
should be fixed now, thank you for your review